frontend/src/components/GlobalAmount/GlobalAmount.test.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+
+import { GlobalAmount } from './GlobalAmount';
+import { GlobalAmountPresentational } from './GlobalAmountPresentational';
+import { GlobalAmountPresentationalProps, GlobalAmountProps } from './Props';
+
+describe('Global Amount component', () => {
+    let props_pres: GlobalAmountPresentationalProps;
```

Comment:
Codacy found an issue: [Identifier 'props_pres' is not in camel case.](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5891730)